### DISCUSSION TITLE
Disabling Password Validation

### DISCUSSION THREAD

**Discussion by pkrakesh** [nrbnlulu/strawberry-django-auth#579] [Q&A] [answered]
Is there any option to disable password validation? I think enforcing strict password validation create a lot of friction for new signing ups.

**Comment by nrbnlulu** [comment, marked as answer]
AFAIR its up to your django auth settings

**Reply by pkrakesh** [reply]
Thank you.

**Reply by pkrakesh** [reply]
I have created a PR to the strawberry-django repo changing its documentation to use this package as the recommended package for JWT Authentication instead of the unmaintained package previously in the documentation . It is approved and merged!!

https://strawberry.rocks/docs/django/guide/authentication#authentication

**Reply by nrbnlulu** [reply]
lol, thanks mate :+1:
